### DISCUSSION TITLE
Mongo DB Persistence

### DISCUSSION THREAD

**Discussion by conductor-community** [Netflix/conductor-community#182] [Ideas]
Similar to the persistence options available - memory, cassandra, mysql etc. Would it be useful to have a Mondo DB Persistence as an option ?

There was a similar discussion [here](https://github.com/Netflix/conductor/discussions/3238)

**Comment by charybr** [comment]
Yes, we are looking for Mongo DB persistence. Peristence can happen in Mongo db and Queueing can happen in orkes-queue (Redis cluster).

There is already a PR for Mongo persistence, which i have not yet tried  - https://github.com/maheshyaddanapudi/conductor/pull/3
